gson/src/test/java/com/google/gson/GsonBuilderTest.java

```diff
@@ -52,20 +143,6 @@ public void testExcludeFieldsWithModifiers() {
     assertEquals("{\"d\":\"d\"}", gson.toJson(new HasModifiers()));
   }
 
-  public void testRegisterTypeAdapterForCoreType() {

```

Comment:
Moved this down to not be between two test method for `GsonBuilder.excludeFieldsWithModifiers`.